src/electionguard_gui/web/components/shared/key-ceremony-list-component.js

```diff
@@ -39,8 +40,13 @@ export default {
   },
   template: /*html*/ `
   <spinner :visible="loading"></spinner>
-  <p v-if="!keyCeremonies">No key ceremonies found...</p>
-  <div v-if="keyCeremonies" class="d-grid gap-2 d-md-block">
+  <div v-if="showWhenEmpty">
+    <p v-if="!keyCeremonies.length">
```

Comment:
Do we need both a showWhenEmpty and a check against keyCeremonies.length being not zero?  Seems like we could use 1 or the other.

Reply:
We do need both, however that's a bit hard to read, I'll combine the `v-if` statements